spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -777,27 +777,39 @@ The Jakarta Data implementation automatically recognizes the query annotations i
 
 === Parameter-based automatic query methods
```

Comment:
The section right before this is named "Annotated Query methods", so let's call this section "Annotated Find methods" now that the `Find` annotation gives us a better way to refer to this pattern. 
```suggestion
=== Annotated Find methods
```